### ISSUE TITLE
Choose accent color

### ISSUE THREAD

**Issue by Uranusek** [Droid-ify/client#611]
Choosing an accent color would be a nice feature. Something like in the Aurora Store.

**Comment by meenbeese** [comment]
I don't think this is planned, as it goes against Material You principles to manually choose the accent color. For example,  Aurora Store has removed this feature after migrating to Material You in full, so I don't see why Droid-ify should implement this.

**Comment by Uranusek** [comment]
Another reason to dislike Material You. I'll stick with the older version.
